docker/safety-rules/run-key-manager.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+# Example Usage (arguments that are not specified will be assigned defaults):
+# ./run-key-manager.sh <IMAGE> <JSON_RPC_ENDPOINT> <ROTATION_PERIOD_SECS> <VALIDATOR_ACCOUNT> <VAULT_HOST> <VAULT_TOKEN>
+
+IMAGE="${1:-libra_safety_rules:latest}"
+JSON_RPC_ENDPOINT="${2:-http://127.0.0.1:8080}"
+ROTATION_PERIOD_SECS="${3:-3600}"
+VALIDATOR_ACCOUNT="${4:-00000000000000000000000000000000}"
+VAULT_HOST="${5:-http://127.0.0.1:8200}"
+VAULT_TOKEN="${6:-root_token}"
+
+docker network create --subnet 172.18.0.0/24 testnet || true
+
+docker run \
+    -e JSON_RPC_ENDPOINT="$JSON_RPC_ENDPOINT" \
+    -e ROTATION_PERIOD_SECS="$ROTATION_PERIOD_SECS" \
+    -e VALIDATOR_ACCOUNT="$VALIDATOR_ACCOUNT" \
+    -e VAULT_HOST="$VAULT_HOST" \
+    -e VAULT_TOKEN="$VAULT_TOKEN" \
+    --ip 172.18.0.2 \
```

Comment:
Give this a different IP so safety-rules and key-manager can be run at the same time.